package.json

```diff
@@ -77,5 +78,10 @@
     "jslint": "eslint $(git diff --name-status master | grep -E '.js$|.vue$' | cut -f 2)",
     "jslint:fix": "yarn run jslint --fix",
     "csslint": "sass-lint -v $(git diff --name-status master | grep -E '.css$|.scss$' | cut -f 2)"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "i18n-tasks normalize"
```

Comment:
And also `"pre-push": "npm run jslint:fix && npm run csslint"` ?

Reply:
great, I'll investigate it

Reply:
I prefer use pre-commit instead of pre-push.

Pre-push:
- Add files with some mistake like bad indentation or vue/order-components
- git commit -m 'foo'
- git push
❌ husky...

Pre-commit:
- Add files with some mistake like bad indentation or vue/order-components
- git commit -m 'foo'
❌ husky...

I use lint-staged in my projects. [I've seen that people also use it with husky.](https://codeburst.io/continuous-integration-lint-staged-husky-pre-commit-hook-test-setup-47f8172924fc)

Reply:
I also prefer per-commit just to avoid doing two commits: one with the wrong files and the next one with the fixed files.